packages/loot-core/src/server/accounts/sync.js

```diff
@@ -206,13 +206,17 @@ async function downloadNordigenTransactions(
     }
 
     const {
-      transactions: { booked },
+      transactions: { booked, pending, all },
       balances,
       startingBalance,
     } = res;
 
     return {
-      transactions: booked,
+      transactions: {
+        booked,
+        pending,
+        all,
+      },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
‏
```suggestion
      transactions: all,
```

Reply:
Exactly what I hoped we would do! Fixed it 👍🏼